docs/linux/zbd-api.md

```diff
@@ -129,13 +119,9 @@ struct blk_zone {
 ```
 
 As indicated in the comments to this data structure definition, the unit that
-is used to indicate the zone start position, size and write pointer position is
-512B sector size, regardless of the actual logical block size of the device.
-Even for a device with a 4KB physical sector, the above zone descriptor fields
-use a 512-byte sector size unit.
+is used to indicate (1) the zone start position, (2) the zone's size and (3) the write pointer position is 512B sector size. This holds true regardless of the actual logical block size of the device. Even for a device with a 4KB physical sector, the above zone descriptor fields use a 512-byte sector size unit.
```

Comment:
There is a bug here, which was already there in the text: "Even for a device with a 4KB physical sector..." should be "Even for a device with a 4KB logical sector" (s/physical/logical).